docs/advanced/primitive.md

```diff
@@ -0,0 +1,47 @@
+# Primitives
+
+The `<primitive />` component is a versatile  is a low-level component in TresJS that allows you to directly use any Three.js object within your Vue application without any abstraction. It acts as a bridge between Vue's reactivity system and Three.js's scene graph.
```

Comment:
is a versatile, low-level
an abstraction

Reply:
is not an abstraction